src/runtime/Makefile

```diff
@@ -197,7 +197,7 @@ DEFVALIDVHOSTUSERSTOREPATHS := [\"$(DEFVHOSTUSERSTOREPATH)\"]
 DEFFILEMEMBACKEND := ""
 DEFVALIDFILEMEMBACKENDS := [\"$(DEFFILEMEMBACKEND)\"]
 DEFMSIZE9P := 8192
-DEFVFIOMODE := guest-kernel
+DEFVFIOMODE := vfio
```

Comment:
What implications does this have to dynamically switch from one to another? Can I still use a separate config file to enable guest-kernel?

Reply:
Yes, you can still change the config file to use `guest-kernel`, this is just the default.  In addition you can override this on a per-pod basis using the `io.katacontainers.config.runtime.vfio_mode` annotation.